Jenkinsfile

```diff
@@ -24,6 +24,22 @@ def workdir_linux = { String gopath ->
 	gopath + workpath_linux
 }
 
+properties([[$class: 'ParametersDefinitionProperty',
+	parameterDefinitions: [
+		[$class: 'StringParameterDefinition',
+			defaultValue: '',
+			description: 'controller-sdk-go commit sha to use in glide.yaml (not updated if empty)',
+			name : 'SDK_SHA'],
+		[$class: 'StringParameterDefinition',
+			defaultValue: '',
+			description: 'controller-sdk-go repo to use in glide.yaml (not updated if empty)',
+			name: 'SDK_GO_REPO'],
+		[$class: 'BooleanParameterDefinition',
+			defaultValue: true,
+			description: 'option to run e2e tests (default: true)',
+			name: 'RUN_E2E']
```

Comment:
echoing my comment at echoing my comment at https://github.com/deis/controller-sdk-go/pull/41/files#r71024488 - where do this and the above env vars come from?